internal/provider/jwk.go

```diff
@@ -471,3 +481,70 @@ func updateJWKS(vals *schema.Set, jwks string, diags diag.Diagnostics) (string,
 	toReturn = string(jsonString)
 	return toReturn, diags
 }
+
+func mapJWKfromDTO(localKeys []interface{}, jwks string) []interface{} {
+
+	var serverKeysMap map[string][]JWKStruct
+	json.Unmarshal([]byte(jwks), &serverKeysMap)
```

Comment:
Check error from Unmarshal? Even if just a panic, it would help track down potential issues...

Reply:
I've changed to report the errors when parsing the JWK string.
The panic scenario I haven't implemented as there is not much to be done other than exit